src/components/badges/blink/index.ts

```diff
@@ -0,0 +1,3 @@
+import blink from './blink';
+
+export default blink;
```

Comment:
1. Guess we need to export the props too
2. Should we also make it an export at the root level `index.ts`?

Reply:
1. Do we need to export them? I think we rarely do.
2. That is how it's currently done but I'm a bit skeptical. Doesn't seem to be the best way. 

If you believe we need to export them, I'll just add them to the index.ts for now.

Reply:
1. I think we should, as there are times you might maybe wanna use a function to generate props for this or other cases.
2. I'm not getting your concern here. I think the `index.ts` in the root is like the single point of truth. We can do anything inside the library, the main thing is the API we return, in this case the `index.ts` is our API